setup.py

```diff
@@ -37,10 +37,10 @@
 ]
 
 localprocessing_require = [
-    "rioxarray",
+    "rioxarray>=0.13.0",
     "pyproj",
-    "openeo_pg_parser_networkx>=2023.1.2",
-    "openeo_processes_dask>=2023.3.0",
+    "openeo_pg_parser_networkx==2023.3.1",
+    "openeo_processes_dask[implementations]==2023.3.2",
```

Comment:
Generally it's bad style to pin versions down to an exact version in setup.py.
It's common practice to use `==`  in requirements.txt, but setup.py should be more liberal and typically just contain  `>=` constraints.

Is there a particular reason to pin down the versions like that?

Reply:
I followed the suggestion of @LukeWeidenwalker, since he said that the next versions could introduce braking changes.

Reply:
@soxofaan I removed the == in setup.py and added a requirements.txt file just with those two packages. Makes sense or do we need to include more in that file?